process/block/preprocess/transactions_test.go

```diff
@@ -466,6 +467,118 @@ func TestTxsPreprocessor_NewTransactionPreprocessorNilTxExecutionOrderHandler(t
 	assert.Equal(t, process.ErrNilTxExecutionOrderHandler, err)
 }
 
+func TestTxsPreprocessor_NewTransactionPreprocessorBadTxCacheSelectionConfig(t *testing.T) {
```

Comment:
We test (verify) the passed config here - in the constructor of the component where the configuration entries are actually read.